offer-manager-domain/src/main/groovy/life/qbic/business/offers/policies/taxes/OutsideGermanyTaxPolicy.java

```diff
@@ -1,23 +1,23 @@
 package life.qbic.business.offers.policies.taxes;
 
 import java.math.BigDecimal;
-import life.qbic.business.persons.affiliation.AffiliationCategory;
 
-public class OutsideGermanyTaxPolicy extends TaxPolicy {
+public class OutsideGermanyTaxPolicy implements TaxPolicy {
 
-  public static OutsideGermanyTaxPolicy of(AffiliationCategory category, String country) {
-    return new OutsideGermanyTaxPolicy(category, country);
+  protected static OutsideGermanyTaxPolicy of(String country) {
+    if (country.equalsIgnoreCase("Germany")) {
+      throw new PolicyViolationException(
+          "Policy must be applied to countries outside of Germany only.");
+    }
+    return new OutsideGermanyTaxPolicy();
   }
 
-  protected OutsideGermanyTaxPolicy(
-      AffiliationCategory targetAffiliationCategory,
-      String country) {
-    super(targetAffiliationCategory, country);
+  private OutsideGermanyTaxPolicy() {
   }
 
   @Override
   public BigDecimal calculateTaxes(BigDecimal value) {
-    return value;
+    return BigDecimal.ZERO;
```

Comment:
value * getVatRatio()

Reply:
why? It is an uncessasary method call, and it is very explicit and easy to read when you check the method implementation.

Reply:
I undertand, but now you have also defined the same value at two places